app/views/content_items/get_involved.html.erb

```diff
@@ -1,155 +1,229 @@
-                  <div class="consultation-attributes" >
-                    <li><% consultation['organisations'].each do |org| %> <%= org["acronym"] %> <% end %></li>
-                    <li><%= t('get_involved.closes', closing_date: date_microformat(consultation['end_date']) ) %></li>
-                    <li class="consultation-view-link"><%= link_to t('get_involved.read_respond'), consultation['link'], class: "govuk-link", aria: {hidden: true} %></li>
-                  </div>
-                </div>
-              <% end %>
-            </div>
-            <%= link_to(t('get_involved.search_all'), get_consultations_link, class: "govuk-link", id: "see-all-link" ) %>
-          </div>
-          <div class="column govuk-grid-column-one-half-from-desktop recent-outcomes">
-            <div class="content">
-              <%= render "govuk_publishing_components/components/heading", { text: t('get_involved.recent_outcomes'), heading_level: 3, font_size: "s", padding: true } %>
-              <% @recent_consultation_outcomes.each do |consultation| %>
-                <div class="document-row consultation-list-row">
-                  <%= link_to consultation['title'], consultation['link'], class: "govuk-link", id: "consultation-title" %>
-                  <div class="consultation-attributes" >
-                    <li><% consultation['organisations'].each do |org| %> <%= org["acronym"] %> <% end %></li>
-                    <li><%= t('get_involved.closed_with_date', closing_date: date_microformat(consultation['end_date']) ) %></li>
-                    <li class="consultation-view-link"><%= link_to t('get_involved.read_respond'), consultation['link'], class: "govuk-link", aria: {hidden: true}, weight: "bold" %></li>
-                  </div>
-              </div>
-              <% end %>
-            </div>
-          </div>
-        </div>
+    </div>
+
+    <%# Attention to closing consultation  %>
+    <%= render "govuk_publishing_components/components/inset_text", {
+    } do %>
+      <div>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: time_until_closure(@next_closing_consultation),
+        heading_level: 3
+      } %>
+      <p class="govuk-body">
+        <%= @next_closing_consultation['title'] %>
+      </p>
+      <%= link_to t('get_involved.read_respond'), @next_closing_consultation['link'], class: "govuk-link" %>
       </div>
-    </section>
+    <% end %>
+  </div>
+</div>
 
-    <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
+<%# Recently opened section %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds govuk-!-padding-bottom-8">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t('get_involved.recently_opened'),
+      padding: true,
+      border_top: 2,
+      font_size: "s",
+      margin_bottom: 4
+    } %>
+    <%= render "govuk_publishing_components/components/document_list", {
```

Comment:
Not an issue but I remember that with the recent work on [topical events](https://www.gov.uk/government/topical-events/cop26) and [world location news](https://www.gov.uk/world/france/news) we have a similar pattern for the heading and document list components but with slightly different formatting -
```
<div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-7">
  <div id="recently-updated">
    <%= render "govuk_publishing_components/components/heading", {
      text: t('latest_feed.title'),
      padding: true,
      border_top: 2,
    } %>
```
In the above example, the spacing is set to `govuk-!-margin-bottom-7` instead of `govuk-!-margin-bottom-8` and the heading component (despite being a h3) has the default size and no `margin_bottom`. Just wondering if it's worth using the same style properties or not?

Reply:
> Just wondering if it's worth using the same style properties or not?

Agreed, this should match as close as possible.  I've updated the Design to [match similar to this page.](https://www.gov.uk/world/uk-and-the-commonwealth)

----

Side thought:

These image grid patterns are making me re-think override use eg `govuk-!-margin-bottom-7`.   How can we make sure all these instances match across repos?  

Maybe this `image_card` is slowly starting to qualify as a component variation: 

```
.gem-c-image-card__image-grid {
 @include govuk-responsive-padding(6);
}
```

Reply:
Yep sounds good. The utility styles are great for one off overrides but if looks like we starting to use this pattern in quite a few places it might be time to add that variant.

Reply:
🤔   - i'll capture this one on the gem to revisit